### PR TITLE
refactor duplicated code to check Debian Stretch into preflight_checks

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

File: roles/preflight_checks/tasks/main.yml [hunk 1]
```diff
@@ -7,3 +7,10 @@
   assert:
     that:
       - engine_api_db_confd_name == engine_api_db_auth_name
+
+- name: Check UTF-8 locale
+  shell: grep 'LANG=.*\.UTF-8"\?$' /etc/default/locale
+
+- name: Re-generate UTF-8 locale
+  become: true
+  shell: ". /etc/default/locale; grep \"^$LANG\" /etc/locale.gen || echo $LANG UTF-8 >> /etc/locale.gen; locale-gen"
```
Comment (sduthil) [inline roles/preflight_checks/tasks/main.yml]: Why do we need to regenerate the locale if we already asserted that it is the default locale?
Also, can we use the `locale_gen` Ansible module?

Comment (fredericlepied) [reply roles/preflight_checks/tasks/main.yml]: I'm removing this part as this not needed anymore.

**Review by sduthil — APPROVED** [review]

**Pull request merged**